Changelog.md

```diff
@@ -25,6 +25,7 @@ Breaking Changes:
    ``sizeof``, ``supports``, ``typedef`` and ``unchecked``.
  * General: Remove assembly instruction aliases ``sha3`` and ``suicide``
  * General: C99-style scoping rules are enforced now. This was already the case in the experimental 0.5.0 mode.
+ * General: Disallow combining hex numbers with unit denominations (e.g. 0x1e wei).
```

Comment:
``This was already the case in the experimental 0.5.0 mode.`` missing ;).

Reply:
Does it make sense to create a section for items that would have this note? This could then be a subsection of ``Breaking changes``

Reply:
Don't we already have a list of changes that used to be experimental 0.5.0 features somewhere? In any case - we did add such a notice to all the other changelog entries, so we should probably stay with it... we can think about reformatting the changelog before the release (doing it now would require every PR to be rebased, etc.)...

Reply:
Good point. Let's keep this for now and perhaps reorganize them before the release.